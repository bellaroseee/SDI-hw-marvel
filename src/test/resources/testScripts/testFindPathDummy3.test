LoadGraph dummy3 dummy3.tsv

FindPath dummy3 Bill_Greyhate Bill_Greyhate

FindPath dummy3 Bill_Greyhate Black_Bill_The_Feared

FindPath dummy3 Bill_Greyhate Peteplank

FindPath dummy3 Bill_Greyhate Pete_Bones