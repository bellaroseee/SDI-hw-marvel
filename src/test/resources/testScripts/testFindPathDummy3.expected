loaded graph dummy3

path from Bill Greyhate to Bill Greyhate:

path from Bill Greyhate to Black Bill The Feared:
Bill Greyhate to Black Bill The Feared via PPS

path from Bill Greyhate to Peteplank:
Bill Greyhate to Admiral Bill via PPS
Admiral Bill to Peteplank via PPS 13

path from Bill Greyhate to Pete Bones:
Bill Greyhate to Admiral Bill via PPS
Admiral Bill to Pete Bones via PPS 1